src/xdocs/writingjavadocchecks.xml.vm

```diff
@@ -23,25 +23,63 @@
 
     <section name="What is Javadoc comment">
       <p>
-      Javadoc comment is multiline comment <code>/*  */</code> that starts with <b>*</b> character and placed above class definition, interface definition, enum definition, method definition or field definition.
+      Javadoc comment is a multiline comment <code>/*  */</code> that starts with the <b>*</b> character and placed above the class definition, interface definition, enum definition, method definition or field definition.
+      If an annotation precedes any of the definitions listed above, then the javadoc comment should be placed before the annotation.
+      If several multiline comments with javadoc identifiers are placed sequentially, only the nearest on top (above) multiline comment with the javadoc identifier will be used.
```

Comment:
> only the nearest on top (above) multiline comment

This sounds funny.
I assume you are trying to say `only the one closest to the definition, right above it, with the javadoc identifier will be used.`

Reply:
@pbludov, please follow simple rule that save a lot of time, please reply to each review point as "done" to let you recheck that you did all and send us clear signal that you are ready for review, and new commit is not WIP.

Reply:
done